code/modules/paperwork/filingcabinet.dm

```diff
@@ -122,21 +120,6 @@
 			return
 	to_chat(user, span_notice("You find nothing in [src]."))
 
-
-/obj/structure/filingcabinet/Topic(href, href_list)
-	if(!usr.canUseTopic(src, BE_CLOSE, NO_DEXTERITY, FALSE, !iscyborg(usr)))
-		return
-	if(href_list["retrieve"])
-		usr << browse("", "window=filingcabinet") // Close the menu
-
-		var/obj/item/P = locate(href_list["retrieve"]) in src //contents[retrieveindex]
-		if(istype(P) && in_range(src, usr))
-			usr.put_in_hands(P)
-			updateUsrDialog()
-			icon_state = "[initial(icon_state)]-open"
-			addtimer(VARSET_CALLBACK(src, icon_state, initial(icon_state)), 5)
```

Comment:
I thought this was still a topic menu did someone save us

Reply:
#64065

Reply:
Yep, someone converted it to tgui, but forgot to remove the old topic

Reply:
praise fucking be